tests/test_backend.py

```diff
@@ -3059,6 +3059,16 @@ def func(input_tensor, new_shape):
 
         self._run_test_case(func, [_OUTPUT], {_INPUT: input_tensor_val, _INPUT1: new_shape_val})
 
+    def test_bfloat(self):
+        x_val = np.array([0, 1, 2], dtype=np.float32)
+        y_val = np.array([3, 4, 5], dtype=np.float32)
+        def func(x, y):
+            x_ = tf.cast(x, tf.bfloat16)
+            y_ = tf.cast(y, tf.bfloat16)
+            s_ = tf.add(x_, y_)
+            return tf.cast(s_, tf.float32, name=_TFOUTPUT)
+        self._run_test_case(func, [_OUTPUT], {_INPUT: x_val, _INPUT1: y_val})
+
```

Comment:
Few questions below:

   * is there a model that is unblocked by this?
   * the test passes with small int values, but are there any values that will cause a failure?